`make doc` broken

When running `make doc`, I get:

`````` src/front/TopLevel.hs:42:25:
    Exception when trying to run compile-time code:
      ENCORE_BUNDLES: getEnv: does not exist (no environment variable)
    Code: (=<<) ((.) stringE init) (($) runIO getEnv "ENCORE_BUNDLES")
    In the splice:
      $((stringE . init)
        =<< (runIO $ System.Environment.getEnv "ENCORE_BUNDLES"))
make: *** [doc] Error 1```
``````

--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,8 @@ SET_DIR=$(RUNTIME_DIR)/set
 FUTURE_DIR=$(RUNTIME_DIR)/future
 ENCORE_DIR=$(RUNTIME_DIR)/encore
 doc: cabal-config
-	make -C doc/encore/
+	export ENCORE_BUNDLES="$(CURDIR)/bundles/" && \
+	make -C doc/encore/ && \
 	cabal haddock --all
 
 dirs: $(INC_DIR) $(LIB_DIR)

